src/frontend/shortlist/src/components/layout/profile/UserPreference.vue

```diff
@@ -0,0 +1,96 @@
+<script>
+import PreferenceDemo from "../../preferences/PreferenceDemo.vue";
+import axios from "axios";
+
+export default {
+  components: {
+    PreferenceDemo,
+  },
+  data() {
+    return {
+      prefs: {},
+      prefupdateflag: false,
+    };
+  },
+  methods: {
+    updateflag() {
+      this.prefupdateflag = !this.prefupdateflag;
+    },
+  },
+  mounted() {
+    axios
+      .post(
+        "https://api.shortlist.nyc/account/metadata",
+        {},
+        {
+          headers: {
+            "x-shortlist-account": "e44278fec4984522b156cae1a8b3981d",
```

Comment:
This is hard coded. It is a good for now. Could you please add a comment for later to add api call?

Reply:
Currently, We are using one user later we will structure for more/every by type.